docs/src/create-apps/dependency-updates.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Automate your dependency updates
+sidebarTitle: Automated dependency updates
+description: Learn how to automate your dependency updates through a source operation. You can even trigger your dependency updates automatically using crons.
+tier:
+  - Elite
+  - Enterprise
+---
+
+Platform.sh allows you to run automated code updates through [source operations](../create-apps/source-operations.md).
+
+For example, you can set up a source operation to run dependency updates on your project. 
+You can then trigger the source operation and therefore your dependency updates automatically using a cron job.
+
+To get notified of every dependency update,
+set up an [activity script](#notifications-through-an-activity-script) or a [webhook](#notifications-through-a-webhook).
+
+## Before you start
+
+You need:
+
+- The [Platform.sh CLI](../administration/cli/_index.md)
+- An [API token](../administration/cli/api-tokens.md) to authenticate the CLI
+  and run a cron job in your app container
+
+## 1. Update your dependencies using a source operation
+
+To facilitate dependency updates in your project,
+set up a source operation in your [application file](../create-apps/_index.md#a-minimal-application)
+depending on your dependency manager:
+
+{{< codetabs >}}
+
++++
+title=Composer
++++
+
+```yaml
+source:
+    operations:
+        update:
+            command: |
+                set -e
+                composer update
+                git add composer.lock
+                git commit -m "Update Composer dependencies"
```

Comment:
Why if there is nothing to update? It will fail, right?

Reply:
```suggestion
                git commit --allow-empty -m "Update Composer dependencies"
```